lib/tasks/ci.js

```diff
@@ -0,0 +1,158 @@
+/* eslint-disable promise/prefer-await-to-then */
+/* eslint-disable promise/prefer-await-to-callbacks */
+
+const fs = require('fs');
+const path = require('path');
+const log = require('fancy-log');
+const findRoot = require('find-root');
+const requestPromise = require('request-promise');
+const request = require('request');
+const B = require('bluebird');
+const os = require('os');
+const octokit = require('@octokit/rest')();
+const _ = require('lodash');
+
+
+const readFile = B.promisify(fs.readFile, {context: fs});
+const writeFile = B.promisify(fs.writeFile, {context: fs});
+
+const ASSET_NAME_REGEXP = /^appium-\S+.zip$/;
+
+const GITHUB_OWNER = 'appium';
+const GITHUB_REPO = 'appium-build-store';
+
+const OUTPUT_INTERVAL = 60000;
+
+const configure = function configure (gulp, opts) {
+  /**
+   * `mocha-parallel-tests` is broken at the moment, so skipped describe blocks fail
+   * the fix is simple, and this patches the error until they fix the package
+   **/
+  gulp.task('fix-mocha-parallel-tests', function () {
+    const root = opts.projectRoot ? opts.projectRoot : findRoot(__dirname);
+    const filePath = path.resolve(root, 'node_modules', 'mocha-parallel-tests', 'dist', 'main', 'util.js');
+
+    return readFile(filePath, {encoding: 'utf8'})
```

Comment:
cannot we use await?

Reply:
We can, now that we enforce Node 8. But it would be a departure from the style of this package.

Reply:
we'll need to do that sooner or later. await is so easier to read

Reply:
Even so, I'd rather do it throughout the package, and keep things consistent. Doing it piecemeal will make it more difficult to do any work at all.

Reply:
I will open a PR with the switch to `await`.